node/src/components/chainspec_loader/chainspec.rs

```diff
@@ -134,6 +134,22 @@ impl HighwayConfig {
                 max = self.maximum_round_exponent
             );
         }
+
+        if self.finality_threshold_fraction <= Ratio::new(0, 1)
```

Comment:
Is that actually possible with `u64` (unsigned). I assume `Ratio` doesn't contain its own "sign", does it?

Reply:
It can be equal to 0. Less, not really (that's why I omitted this part of the check for the rewards multiplier below). I can change that to `==` if you think it makes more sense.

Reply:
Ah, of course! :+1: